.github/workflows/codeql-analysis.yml

```diff
@@ -51,6 +51,10 @@ jobs:
         # Prefix the list here with "+" to use these queries and those in the config file.
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
+    - uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.0.100'
+
```

Comment:
Does this need to be here?

Reply:
Hmm good catch.For the Preview version it was needed for sure because the autobuild of codeql does take the latest stable version. I'll check quickly

Reply:
Yes it is needed @chrissainty 
I removed it from one of my projects just to test see [here](https://github.com/linkdotnet/Blog/runs/4199398862?check_suite_focus=true)

Also check the offical tracker: [here](https://github.com/github/codeql-action/issues/757)

Currently they are only supporting .NET 5.
Only workaround is the global.json